[CUBRIDMAN-73] Support User Schema. (#324)
CUBRID/cubrid-manual: en/sql/user_schema.rst

***********
User Schema
***********

A schema is a logical collection of database objects. An object can only exist in one schema and cannot exist in another schema at the same time. A schema cannot be created by a user, and when a database user is created, the user owns a single schema with the same name as the user name. A schema contains tables, serials, and triggers. An object in a schema must have a unique name within the same schema, but it can have a name that is duplicated with an object in another schema. Table, serial, and trigger names are prefixed with the schema name, so the schema can be identified by the name.

To access the object, the user must use the path expression of "schema_name.table_name". If the user's schema is the same as the object's schema (owner), the schema name can be omitted from the path expression. If the schema name is omitted, the user's schema name is used. Even if the path expression of “schema_name.table_name” is used, the user must have permission to use the object.

.. note::

>     Since version 11.2, "Schema ≒ Database" has been changed to "Schema ≒ User". Before version 11.2, there was only a single schema in the database. Since version 11.2, users are used as a single schema. It is not possible to access a table owned by another user only by the table name. Tables can be accessed using the schema (owner) name as a prefix.

Reviewer: ```suggestion
    Since version 11.2, "Schema ≒ Database" has been changed to "Schema ≒ User". Before version 11.2, there was only a single schema in the database. Since version 11.2, schema is available for each user. It is not possible to access a table owned by another user only by the table name. Tables can be accessed using the schema (owner) name as a prefix.
```

Author: Thanks, I'll change it.
